src/modules/whitelist/whitelist.md

```diff
@@ -93,4 +93,12 @@ the remaining public keys are verified out-of-band when setting up the system,
 so there is no direct benefit to this. We do it only to reduce fragility and
 increase safety of unforeseen uses.
 
-
+Having to access the offline key `Q_i` to compute the secret to the sum `W +
+Q_i` for every authorization is onerous.  Instead, if the whitelisted keys are
+created using
+[BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki)
+unhardened derivation, the sum can be computed on an online machine. In order
+to achieve that, the offline key `Q_j` is set to the negated BIP32 master
```

Comment:
Not the master key, but the last hardened derivation parent key(typically, the pubkey corresponding to the `xpub`?

Reply:
thanks, fixed